lepidopter-fh/setup-ooniprobe.sh

```diff
@@ -1,13 +1,35 @@
 #!/bin/bash
+set -ex
+
+TOR_DEB_REPO="http://deb.torproject.org/torproject.org"
+TOR_DEB_REPO_SRC_LIST="/etc/apt/sources.list.d/tor.list"
+TOR_REPO_GPG="A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89"
+
+# Add Torproject Debian repository
+apt-key adv --keyserver hkp://pool.sks-keyservers.net --recv-keys ${TOR_REPO_GPG}
+echo "deb ${TOR_DEB_REPO} ${DEB_RELEASE} main" > ${TOR_DEB_REPO_SRC_LIST}
+
 # Install golang-go
 apt-get update
-apt-get -y -t stretch install golang-go
+apt-get -y -t ${DEB_RELEASE}-backports install golang-go
```

Comment:
jessie-backport repo [version](https://packages.debian.org/jessie-backports/golang-go): 2:1.5.1-4~bpo8+1
stretch repo [version](https://packages.debian.org/stretch/golang-go): 2:1.5.1-4
I don't think that there is a version difference (maybe minor differences), and installing a package from testing (stretch) with many dependencies is not really nice for the rest of the package dependencies in case of upgrades.